dockers/docker-lldp/start.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+if [ "${RUNTIME_OWNER}" == "" ]; then
+    RUNTIME_OWNER="kube"
+fi
+
+/usr/share/sonic/scripts/container_state.py up -f lldp -o ${RUNTIME_OWNER} -v ${IMAGE_VERSION}
```

Comment:
inside a container, you have two script container_state and container_monitor? both seems deal with container state, why not combine both?

Reply:
They are for two very different purposes.

container_state.sh --
- A synchronous script that checks and updates container state, for both up & down
- It might *block* as needed, during "up".
- Called from start.sh as post-start of the container to mark "up". Start.sh blocks until this script returns.
- Called from container_monitor, upon pre-exit of the container to mark "down". 
- Not a daemon, but a synchronous script

container_monitor:
- A ***daemon*** that sleeps on SIGTERM -to monitor container exit
- Upon SIGTERM, call container_state to mark "down"